app/serializers/web_hook_topic_view_serializer.rb

```diff
@@ -0,0 +1,221 @@
+require_dependency 'pinned_check'
```

Comment:
It's a copy from `TopicViewSerializer` without the post stream so it's much less (could be 1/3 of original size).